tests/iam/unit_tests/test_filters.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Compilation of filters for testing purposes."""
+
+from datetime import datetime
+from google.cloud.security.iam.explain.filters import TimeFilter
+
+def create_time_filter(start_from, end_at, list_untimed_resources):
```

Comment:
ditto: please document the parameters and return

You can see examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html